server/views/components/actions-card/_inductionActions.njk

```diff
@@ -40,7 +40,7 @@
     <ul class="govuk-list govuk-!-margin-0" data-qa="induction-action-items">
       {% if params.inductionSchedule.inductionStatus == 'ON_HOLD' %}
 
-        {% if params.hasEditAuthority %}
+        {% if params.userHasPermissionTo('REMOVE_INDUCTION_EXEMPTION') %}
```

Comment:
And this is how we use the new `userHasPermissionTo` view model function.
Remove the previous role based check, replacing it with a clear and expressive check whether you have permission to do a named thing 👍